vega_sim/scenario/curve_market_maker/scenario.py

```diff
@@ -220,6 +222,18 @@ def set_up_background_market(
             opening_auction_trade_amount=self.opening_auction_trade_amount,
             tag=str(tag),
         )
+
+        info_trader = InformedTrader(
+            wallet_name = INFORMED_WALLET.name,
+            wallet_pass = INFORMED_WALLET.passphrase,
+            price_process = price_process,
+            market_name = market_name,
+            asset_name = asset_name,
+            initial_asset_mint = self.initial_asset_mint,
+            # proportion_taken = self.proportion_taken,
```

Comment:
This should be passed in as one of the many many parameters to this scenario 😄

Reply:
how should I set it up? I used `self.proportion_taken `but its not happy about it